File: Source/Rio.Models/DataTransferObjects/ParcelAllocation/ParcelAllocationDto.cs
namespace Rio.Models.DataTransferObjects.ParcelAllocation
{
    public class ParcelAllocationDto
    {
        public int ParcelID { get; set; }
        public int WaterYear { get; set; }
        public int WaterTypeID { get; set; }
        public decimal AcreFeetAllocated { get; set; }
        public int ParcelAllocationID { get; set; }
    }
}

File: Source/Rio.Web/src/app/shared/models/parcel/parcel-allocation-dto.ts
export class ParcelAllocationDto {
    ParcelID: number;
    WaterYear: number;
    WaterTypeID: number;
    AcreFeetAllocated: number;
    ParcelAllocationID: number;

    constructor(obj?: any) {
        Object.assign(this, obj);
    }
}